src/components/ui/error-text.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import {StyleSheet, TextProps} from 'react-native';
+
+import {Text} from './text';
+
+import {LIGHT_TEXT_BASE_3, LIGHT_TEXT_RED_1} from '../../variables';
+
+type ErrorTextProps = {
+  e0?: boolean;
+  e1?: boolean;
+  e2?: boolean;
+  e3?: boolean;
+} & TextProps;
+
+export const ErrorText = ({
+  e0,
+  e1,
+  e2,
+  e3,
+  style,
+  ...props
+}: ErrorTextProps) => {
+  return (
+    <Text
+      t6={e0}
+      t14={e1 || e2}
+      t8={e3}
+      style={[
+        e0 && StyleSheet.flatten([page.e0Style, style]),
+        e1 && StyleSheet.flatten([page.e1Style, style]),
+        e2 && StyleSheet.flatten([page.e2Style, style]),
+        e3 && StyleSheet.flatten([page.e3Style, style]),
+      ]}
+      {...props}
+    />
+  );
+};
+
+const page = StyleSheet.create({
+  e0Style: {
+    fontStyle: 'normal',
+    fontSize: 16,
```

Comment:
what is the fontSize and lineHeight for here?

Reply:
This is due to the fact that text t6 uses the required text font and weight, but the line height and font size are not suitable
<img width="293" alt="image" src="https://user-images.githubusercontent.com/6774813/201912926-2f051007-4e09-4c5b-a2ad-5b1bc94b3aa5.png">

Reply:
there should not be a font size and line height, all these parameters should be described in the text, if there are discrepancies, then you need to ask the design to update everything in itself and, if necessary, add a new key to the text component